### PR TITLE
Story/ccp 78

### PR DESCRIPTION
**Pull request by mkarthik90**
[CCP-78] Navigated to delete screen when user clicks on delete button in concept search screen. Included tooltip for wordnet concepts as per comments.

### CODE REVIEW

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
I still don't see a different between wordnet concepts (button should be disabled or not there) and delete button for concepts that can be deleted:
image

**Comment by mkarthik90** [comment]
@jdamerow  I tried to grey out the edit and delete and I got the output as in the below image. I am not sure if this is what you are expecting. I am not able to grey out the links completely.

**Comment by mkarthik90** [comment]
@jdamerow  Pls let me know if the below image is fine. I have grayed out image using css. I have committed this code to branch

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/bean/ConceptEditBean.java [hunk 1]
```diff
@@ -188,5 +176,17 @@ public void setFromHomeScreen(boolean fromHomeScreen) {
...
+        posMap.put(POS.VERB, "Verb");
+        posMap.put(POS.ADVERB, "Adverb");
+        posMap.put(POS.ADJECTIVE, "Adjective");
+        return posMap;
+	}
+
+	public void setPosMap(Map<String, String> posMap) {
+		this.posMap = posMap;
+	}
+
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/bean/ConceptEditBean.java]: formatting is all over the place

**Comment by jdamerow** [comment]
retest this please

**Comment by jdamerow** [comment]
please just correct the formatting and then I'll merge it

**Comment by jdamerow** [comment]
please merge conflicts

**Comment by mkarthik90** [comment]
@jdamerow  Rebased the branch with develop

**Comment by jdamerow** [comment]
retest this please

**Pull request merged**